packages/sit-onyx/src/components/examples/GridPlayground/GridPlayground.ct.tsx

```diff
@@ -0,0 +1,146 @@
+import { expect, test } from "../../../playwright/a11y";
+import { ONYX_BREAKPOINTS } from "../../../types";
+import GridPlayground from "./GridPlayground.vue";
+
```

Comment:
would it be a good place to add screenshots to the actual configs, e.g. that our grid works with the centered layout etc or do we already have that kind of tests for the grid, do you know? (lost track with all those screenshots ^^)

Reply:
We already have that but I can check if we can maybe move this to the grid playground in a follow up PR :)

Reply:
E.g. in `packages/sit-onyx/playwright/snapshots/components/OnyxNavBar/grid-max-width-xl-chromium-linux.png`

Reply:
aah ok! 
well, the playground looks better :D